src/risedevtool/src/task/ensure_stop_service.rs

```diff
@@ -31,6 +31,10 @@ impl Task for EnsureStopService {
         ctx.service(self);
 
         for (port, service) in &self.ports {
+            // Do not require stopping kafka services
+            if service.starts_with("kafka") {
+                continue;
+            }
```

Comment:
Why do we need this change?

Reply:
It doesn't make sense to require us to stop user-managed kafka before starting `risedev d`. Even for risedev managed kafka, I guess it's also unnecessary.

Reply:
Shall we apply this to all user-managed or external services? Hardcoding to `kafka` does not look elegant. 😄